omod/src/main/resources/apps/patient_dashboard_app.json

```diff
@@ -24,6 +24,29 @@
     ]
   },
   {
+    "id": "ugandaemr.dashboardwidget.MCHSummary",
+    "instanceOf": "coreapps.template.dashboardWidget",
+    "description": "coreapps.latestObsForConceptList.app.description",
+    "order": 10,
+    "config": {
+      "widget": "latestobsforconceptlist",
+      "icon": "icon-user-md",
+      "label": "MCH Summary",
+      "concepts": "dcc033e5-30ab-102d-86b0-7a5022ba4115"
```

Comment:
There is only one concept here, you need to add a comma delimited list of concepts whose information is to be shown in the order that they are to be shown